__tests__/auth.test.ts

```diff
@@ -71,7 +105,11 @@ describe('auth tests', () => {
     const password = 'TOKEN';
     const gpgPassphrase = 'GPG';
 
-    await auth.configAuthentication(id, username, password, gpgPassphrase);
+    commonObject.id = id;
+    commonObject.username = username;
+    commonObject.password = password;
+    commonObject.gpgPassphrase = gpgPassphrase;
+    await auth.configureAuthentication();
```

Comment:
such kind of tests looks a bit unclear. let's move auth back to the separate method to simplify testing

Reply:
Done. Thank you.